src/components/Search/Search.jsx

```diff
@@ -11,20 +11,23 @@ type Props = {|
   iconUrl: string,
   instructionText: string,
   onChange: ({target: {value: string}}) => void,
+  onKeyDown: (event: Event) => void,
```

Comment:
doesnt this need to be nullable?

Reply:
ie. onKeyDown?: (event: Event) => void

Reply:
I'm guessing this works because we have it in default props